src/test/java/bio/terra/common/sam/SamRetryTest.java

```diff
@@ -17,7 +17,7 @@
 @Tag("unit")
 public class SamRetryTest {
 
-  private static Logger logger = LoggerFactory.getLogger(SamRetryTest.class);
+  private static final Logger logger = LoggerFactory.getLogger(SamRetryTest.class);
```

Comment:
aside: Is it better for loggers to be static? Many of the ones in wsm are not today.